packages/badge/src/IndicatorDot.tsx

```diff
@@ -1,23 +1,24 @@
 import { Box } from '@ag.ds-next/box';
-import { BadgeTone, badgeToneMap } from './utils';
+import { boxPalette } from '@ag.ds-next/core';
 
 export type IndicatorDotProps = {
 	/** The accessible label to read out in screen readers. */
 	'aria-label'?: string;
-	/** The colour tone to apply. */
-	tone: BadgeTone;
 };
 
 export const IndicatorDot = ({
 	'aria-label': ariaLabel,
-	tone,
 }: IndicatorDotProps) => {
-	const backgroundColor = badgeToneMap[tone];
 	return (
 		<Box
 			aria-label={ariaLabel}
 			highContrastOutline
-			css={{ width: 8, height: 8, borderRadius: 4, backgroundColor }}
+			css={{
```

Comment:
Suggestion: these styles (apart from `backgroundColor`) are duplicated in StatusBadge.
Consider putting them in an object that you can export so styles are less able to drift.

Reply:
I considered this but I didn't want to expose `successDotStyles` to consumers. It's more of a coincidence that these things share appearance.